textbook/11/2/experiments.ipynb

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
<p>Is the only change here that the contents of the markdown file were put into a single markdown cell in a notebook?</p><p> </p><p>If it helps: I would say that there is <em>not</em> a preference for notebook files. If this section isn't intended to have interactive/interpreted code, it should just be a markdown file. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yes that is the only change, we wanted notebooks since they work with reviewNB. We cant open md docs with reviewNB